kotsadm/web/src/components/gitops/GitOpsDeploymentManager.jsx

```diff
@@ -261,12 +290,11 @@ class GitOpsDeploymentManager extends React.Component {
       return;
     }
 
-    const getGitOpsRepo = this.props.getGitOpsRepoQuery?.getGitOpsRepo;
-    if (!getGitOpsRepo) {
+    if (!this.state.gitops) {
```

Comment:
since gitops initial value is `{}`, this will never be true, you can use lodash `isEmpty` function

Reply:
example: https://github.com/replicatedhq/kots/blob/bb2be4d67d912a0bdb074c9f2cafbff2dd3d6d2c/kotsadm/web/src/components/apps/AppSnapshotDetail.jsx#L7